ide/api.lsp/manifest.mf

```diff
@@ -1,5 +1,5 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.api.lsp/1
 OpenIDE-Module-Localizing-Bundle: org/netbeans/api/lsp/Bundle.properties
-OpenIDE-Module-Specification-Version: 1.28
+OpenIDE-Module-Specification-Version: 1.29
```

Comment:
We can't really do spec version increments on delivery at the moment.  If this gets merged to delivery it should still use `1.28`